Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:18.17.0-alpine
+
+WORKDIR /app
+
+# Update npm
+RUN npm install -g npm@latest
+
+# Install app dependencies
+COPY ["package.json", "package-lock.json*", "yarn.lock*", "./"]
```

Comment:
the project is solemnly dependent on yarn so you can remove package-lock.json

Reply:
Okay I will remove the package-lock dependency

Reply:
package-lock.json removed from the dependency list